openedx/core/djangoapps/catalog/utils.py

```diff
@@ -112,7 +123,7 @@ def get_programs(site=None, uuid=None, uuids=None, course=None, catalog_course_u
         organization (string): short name for specific organization to read from the cache.
 
     Returns:
-        list of dict, representing programs.
+        list of str, representing programs.
```

Comment:
comment change just reflects what I believe was the state of affairs. I do not believe that this returns a list of dicts.